.pre-commit-config.yaml

```
@@ -151,59 +151,16 @@ repos:
   rev: v2.1.0
   hooks:
   - id: codespell
+    # FIXME: Address remove current spelling exceptions:
+    args: ["-L", "tipe,dout"]
```

Let's be consistent in the sequence style, plus explain what these are:
```suggestion
    args:
    - -L
    - tipe,dout  # vars currently present in the codebase
```